### PR TITLE
Syndie Space Rock Ghost Role and Radio Station

### PR DESCRIPTION
**Pull request by AshTheDerg**
# About The Pull Request

Adds a Lone Syndicate Comms agent, but in space, featuring a smaller map for better spawn chance. Also adds a Radio Station. Both SHOULD naturally spawn. Hopefully.

## Why It's Good For The Game

More Ghost Role, and Radio Station for RP Purposes.

## A Port?

No, just a redone Map

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added Syndicate Asteroid Listening Post + Ghost Role
add: Added Radio Station Ship
/:cl:

### CODE REVIEW

**Comment by Yawet330** [comment]
The syndie comms agent in space already exists.

**Comment by AshTheDerg** [comment]
> The syndie comms agent in space already exists.

It's blacklisted, believe it or not. The map is also unnecessarily massive.

File: modular_splurt/code/game/area/areas/ruins/space.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "DJ"
+	has_gravity = STANDARD_GRAVITY
+
+/area/ruin/space/radiostation/cabin
+	name = "NT Radio Cabin"
+	icon_state = "dorms"
+	has_gravity = STANDARD_GRAVITY
+
+/datum/map_template/ruin/space/radiostation
+	id = "radiostation"
```
Comment (BongaTheProto) [inline modular_splurt/code/game/area/areas/ruins/space.dm:41]:
```suggestion
/datum/map_template/ruin/space/radiostation
	prefix = "modular_splurt/_maps/RandomRuins/SpaceRuins/"
	id = "radiostation"
```

File: modular_splurt/code/game/area/areas/ruins/space.dm [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	description = "A mobile NT Radio Station ship."
+
+
+/area/ruin/space/syndielistenspace
+	name = "Syndicate Listening Post"
+	icon_state = "yellow"
+	has_gravity = STANDARD_GRAVITY
+
+/datum/map_template/ruin/space/syndielistenspace
+	id = "syndielistenspace"
```
Comment (BongaTheProto) [inline modular_splurt/code/game/area/areas/ruins/space.dm:53]:
```suggestion
/datum/map_template/ruin/space/syndielistenspace
	prefix = "modular_splurt/_maps/RandomRuins/SpaceRuins/"
	id = "syndielistenspace"
```

**Comment by AshTheDerg** [comment]
> This actually can create duplicate space syndicate bases since this doesnt' disable the previous comms agent outpost, which has a 1/6 chance (IIRC) to spawn with a syndicate sleeper in it. image I recommend disabling the default one as yours is superior.
> 
> Some of your changes too seem to have just been map-merged out of reality due to an accident. May want to investigate this, since the only difference is a slightly larger rock between the two

The original is 100% commented out, it _can't_ spawn, as it doesn't exist to the code.

**Pull request merged**